nook/functions/common/python/claude_client.py

```
@@ -0,0 +1,220 @@
+            "top_p": top_p or self._config.top_p,
+            "max_tokens": max_output_tokens or self._config.max_output_tokens,
+        }
+
+        if system_instruction:
+            api_params["system"] = system_instruction
+
+        try:
+            response = self._client.messages.create(**api_params)
+            return response.content[0].text
+        except Exception as e:
+            logger.error(f"Error generating content with Claude: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)